src/components/MyKiva/RoundCheckbox.vue

```diff
@@ -26,11 +26,13 @@
 				]"
 				style="aspect-ratio: 1 / 1;"
 			>
-				<KvMaterialIcon
-					v-if="inputValue"
-					:icon="mdiCheckCircle"
-					class="tw-w-3.5 tw-h-3.5 tw-text-brand-550"
-				/>
+				<transition name="fade">
```

Comment:
Animations look good! Can we start the animations when the lending stats component enters the viewport? Not important for desktop, but it would probably needed for mobile, right?